### ISSUE TITLE
Basin information panel

### ISSUE THREAD

**Issue by fmaussion** [finnwimberly/glacial_runoff_mapping#1]
Thanks a lot for the app, I think this is really useful for outreach and exploring the data. After using it and showing it to a few people, I noted a few ideas that could improve it further.

<img width="900" alt="image" src="https://github.com/finnwimberly/glacial_runoff_mapping/assets/10050469/0edeb832-e854-4631-9961-793cfe7e7d71">

- [x] "Initial Ice Volume" -> "Ice volume in 2000"
- [ ] "Glacier area in 2000" is missing
- [ ] "Basin area" is missing
- [x]  "Percent Area Initially Glaciated" ->  "Glaciated area (%) in 2000"
- [ ] "Year of Peak Water" -> it's unclear to which model it refers (probably multi model median), and is there perhaps a way to add uncertainty?

**Comment by lilianschuster** [comment]
I agree with @fmaussion  that it would be great to have  "Glacier area in 2000", "Basin area" and to change to "Multi-model median Year of Peak Water".
